stacks/dlaas/oss/README.md

```diff
@@ -0,0 +1,25 @@
+# Clear Linux Deep Learning Stack
+[![](https://images.microbadger.com/badges/image/clearlinux/dlaas.svg)](http://microbadger.com/images/clearlinux/dlaas "Get your own image badge on microbadger.com")
+
+This provides a Clear Linux Deep Learning Stack. To offer more flexibility, there are two versions of the Deep Learning Stack: Tensorflow* framework optimized for Intel Architecture and a second version that includes TensorFlow* framework optimized using Intel® Math Kernel Library for Deep Neural Networks (Intel® MKL-DNN) primitives.
+
```

Comment:
I would add something like "This version relates to the former"

Reply:
hrm. let me see how to word it maybe put that alone in the top level readme instead of here?

Reply:
agree. this would make it even more clear.